database_setup.rb

```diff
@@ -0,0 +1,55 @@
+require 'pg'
+
+def setup_database
+  begin
+    connection=PG.connect(dbname: 'postgres' , user: 'lakshayjain' , host: 'localhost')
```

Comment:
Please use space on both side of operators. I'll soon share the style guide the covers most of the things. But for now, I'll share few points on PR as they come so that you're not overwhelmed.

Reply:
Also, should I expect "lakshayjain" user to exist on my machine? I think this is hardcoded as per your system. See how we can pass these details from outside the application